slt-app/package.json

```diff
@@ -16,6 +16,8 @@
     "web-vitals": "^1.0.1"
   },
   "scripts": {
+	"server:install": "cd api && conda create -f environment.yml && and cd ..",
```

Comment:
Also, when trying to run the correct script, i get `ResolvePackageNotFound:` doing some googling, I think that its because some of the packages that you installed (full list below) have a checksum (py36h...) thats only for windows, not OSX. 

Solution: when creating the environment.yml do this instead
`conda env export --no-builds > environment.yml`

from: https://github.com/conda/conda/issues/6073#issuecomment-356981567